lib/xdr/ledger_entries/trust_line_entry.ex

```diff
@@ -0,0 +1,143 @@
+defmodule StellarBase.XDR.TrustLineEntry do
+  @moduledoc """
+  Representation of Stellar `TrustLineEntry` type.
+  """
+  alias StellarBase.XDR.{AccountID, TrustLineAsset, Int64, UInt32, Ext}
+
+  @behaviour XDR.Declaration
+
+  @struct_spec XDR.Struct.new(
+                 account_id: AccountID,
+                 asset: TrustLineAsset,
+                 balance: Int64,
+                 limit: Int64,
+                 flags: UInt32,
```

Comment:
@einerzg Thanks for contributing

There is a missing union called `v1` who at the same time holds another struct construction. Details in:  https://github.com/stellar/stellar-core/blob/master/src/xdr/Stellar-ledger-entries.x#L274

These inline constructions need to be handled as separated XDR types. You can create them as:
* `TrustLineEntryV1` 
* `TrustLineEntryExt`

Best!!